test_outputs.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+declare -a goodfiles
+declare -a outputfiles
+
+for filename in good_outputs/*.txt; do
+	goodfiles+=($filename)
+done
+
+for filename in outputs/*.txt; do
+	outputfiles+=($filename)
+done
+
+differences=0
+
+for goodfile in goodfiles
+do
+	for outputfile in outputfiles
+	do
+		if [ "$goodfile" = "$outputfile" ]; then
+			otherdiffs=`diff goodfile outputfile`
+			differences=$((differences+otherdiffs))
+		fi
+	done
+done
```

Comment:
Ezt talan kicsit szebben is meglehetne. Nem azt, hogy megnezem minden fajlra, hogy ugyanazok, hanem megnezem a generalt fajlokat, es ugyanazokat a fajlneveket megkeresem a good outputok kozott, hogy ott van-e. Ha nem, akkor a diff-be bele irok valami errort, egyebkent megdiffelem.

Reply:
Ár írtuk a test scriptet, a units/good_outputs mappában található fájlokon meg végig és azokhoz hasonlítja az ugyan olyan nevű fájlokat az test/outputs mappából.